google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -317,13 +317,17 @@ public DatabaseClient getDatabaseClient(DatabaseId db) {
         return dbClients.get(db);
       } else {
         SessionPool pool = SessionPool.createPool(getOptions(), db, SpannerImpl.this);
-        DatabaseClientImpl dbClient = new DatabaseClientImpl(pool);
+        DatabaseClientImpl dbClient = createDatabaseClient(pool);
         dbClients.put(db, dbClient);
         return dbClient;
       }
     }
   }
 
+  DatabaseClientImpl createDatabaseClient(SessionPool pool) {
```

Comment:
Please make these types of style changes in a different PR.

Reply:
I'll keep that in mind for future changes.
For now, I added a missing `@VisibleForTesting` annotation.